### ISSUE TITLE
RM44 weight incorrect

### ISSUE THREAD

**Issue by BatuKhan** [AthenaSulisMinerva/CombatExtendedFastTrack#226]
The RM44 (Magnum) has a weight of 92 kg. I can only assume this must be an error.

**Comment by AthenaSulisMinerva** [comment]
Please follow the [issue posting rules](https://github.com/AthenaSulisMinerva/CombatExtendedFastTrack/issues/175) and provide your game and mod version information.

Which mod is this "RM44" weapon from?

**Comment by skycrossercat** [comment]
@AthenaSulisMinerva Just about to post the same thing XD.

**Comment by BatuKhan** [comment]
> Please follow the [issue posting rules](https://github.com/AthenaSulisMinerva/CombatExtendedFastTrack/issues/175) and provide your game and mod version information.

Oh sorry. Game v. 1.0.2408, CE FT v. 1.0.1.6.

> Which mod is this "RM44" weapon from?

I'm not sure, I thought it was a part of CE. The only other weapon mods I have are Vanilla Weapons Expanded and Vanilla Weapons Expanded - Quickdraw, but it's not  a part of those mods. I also have the compatibility patch for those mods.

**Comment by sumghai** [comment]
Yeah, we're gonna need the exact name of the mod and the defName of the weapon in question.

**Comment by BatuKhan** [comment]
> Yeah, we're gonna need the exact name of the mod and the defName of the weapon in question.

So seems it's the Hand Cannon from CE.

https://i.imgur.com/Txw0xg4.png

**Comment by sumghai** [comment]
> So seems it's the Hand Cannon from CE.

Erm, according to the screenshot, the defName of that gun is `VWE_Gun_HandCannon` - VWE stands for **Vanilla Weapons Expanded**, not Combat Extended.

Also, CE:FT does not come with built-in support for VWE, so this implies that you are, in fact, running [the standalone third-party VWE CE patch mod](https://steamcommunity.com/sharedfiles/filedetails/?id=1819425140), which CE:FT is not responsible for:

Furthermore, this is a known issue on the VWE CE Workshop page:

![Annotation 2019-12-30 192046](https://user-images.githubusercontent.com/4993380/71579951-aafc1980-2b39-11ea-85e2-e9d179d8950d.png)

**In Summary:** VWE CE needs to fix their patches, not us.
